docs/page-objects/testing.rst

```diff
@@ -202,3 +202,24 @@ Please also check the official Git LFS documentation for more information.
 
 .. _Git LFS: https://git-lfs.com/
 .. _implementations: https://github.com/git-lfs/git-lfs/wiki/Implementations
+
+Additional requests support
+===========================
+
+If the page object uses the :class:`~.HttpClient` dependency to make
+:ref:`additional requests <additional-requests>`, the generated fixtures will
+contain these requests and their responses. When the test runs,
+:class:`~.HttpClient` will return the saved responses without doing actual
+requests.
+
+Currently requests are compared by their URL, method, headers and body, so if a
+page object makes requests that differ between runs, the test won't be able to
+find a saved response and will fail.
+
+Test coverage
+=============
+
+The coverage for page object code is reported correctly if tools such as
+`coverage`_ are used when running web-poet tests.
+
+.. _coverage: https://coverage.readthedocs.io/
```

Comment:
Do we need to cover this in the docs? To me it feels like a given; if it was not the case before, this could be covered in the changelog.

Reply:
@kmike suggested adding it :)

Reply:
We had this question asked, so maybe it's not given for the users. 
I'm fine both with keeping and dropping it.

Reply:
> We had this question asked.

I guess that is motivation enough. :+1: